src/client/components/filter-menu/boolean-filter-menu/boolean-filter-menu.scss

```diff
@@ -25,6 +25,7 @@ $checkbox-width: 20px;
     @extend %white-shadow;
     background: $white;
     color: $text-standard;
+    height: 80px;
```

Comment:
some values are variables (colors for example), maybe height should be variable too?